Documentation/profiles.md

```diff
@@ -0,0 +1,108 @@
+# Container Profiles
+
+The security of a container is influenced by its configuration. They are
+several parameters:
+
+- *Namespaces:* each kind of namespace (cgroup, ipc, mnt, net, pid, user, uts)
+  could be shared with the host or unshared. Some namespaces can have their own
+  configuration (e.g. uid mappings for user namespaces).
+- *User:* root or non-root
+- *Cgroups:* each kind of cgroup subsystem can have specific configuration
+  (e.g. `devices` cgroup subsystem)
+- *seccomp:* some system calls are restricted in the container
+- *Linux Security Modules (LSMs):* AppArmor, SELinux
+- *Volumes:* e.g. a host volume
+
+Offering a test environment for all possible configuration would be a
+combinational explosion. Instead, the Container Escape Bounty project defines a
+few _container profiles_ with predefined configurations. Security researchers
+can then start containers under one container profile. The bounty reward could
+be different depending on which container profiles the vulnerability is
+exploitable on.
+
+Note that the container profiles are not implemented yet.
+
+The basic container profiles are the following:
+- *Default Docker*
+- *Weak Docker*
+- *Docker with User Namespaces*
+
+Each basic container profiles have the following variants:
+- *without LSM*
+- *with SELinux*
+- *with Apparmor*
+
+## Description of each container profile
+
+### Default Docker
+
+- Namespaces
+  - cgroup
+  - ipc
+  - mnt
+  - net
+  - pid
+  - user
+  - uts
+
+TODO
```

Comment:
do you plan to add these todos over the course of time later or in the same PR ?

Reply:
I was thinking over the course of time later.

Reply:
cool LGTM